images/build/debian-base/Makefile

```diff
@@ -26,7 +26,6 @@ TAR_FILE ?= rootfs.tar
 ARCH ?= amd64
 ALL_ARCH = amd64 arm arm64 ppc64le s390x
 
-TEMP_DIR:=$(shell mktemp -d)
 QEMUVERSION=5.2.0-2
 
 SUDO=$(if $(filter 0,$(shell id -u)),,sudo)
```

Comment:
Is this needed?

Reply:
Removed because it does not seem used anywhere :+1:

Reply:
I think I saw it in debian-iptables Makefile as well. But that can be a separate PR.